### DISCUSSION TITLE
Pinecone Export Failing due Meta Values

### DISCUSSION THREAD

**Discussion by vatsal2210** [AI-Northstar-Tech/vector-io#63] [Q&A]
While exporting from Pinecone, it seems like due to Meta Data, it is failing in export. 

Is there any way to either throw an error at the end OR fix this issue? 

Copied the error message below:

``` 
File "pyarrow/error.pxi", line 91, in pyarrow.lib.check_status
pyarrow.lib.ArrowTypeError: ("Expected bytes, got a 'float' object", 'Conversion failed for column speaker with type object')
```

**Comment by dhruv-anand-aintech** [comment]
Thanks for reporting. Would you be able to provide a sample dataset that is in the index, maybe with just a few rows, so that I can understand the type of data where it fails?

Also, can you tell me which version of the library you're on: `pip show vdf-io`
